modules/administration_manual/pages/configuration/files/external_storage/dropbox.adoc

```diff
@@ -58,7 +58,7 @@ However, the internal Dropbox functionality should be removed in ownCloud 10.0.4
 
 Then, you need to provide a name for the folder in the "Folder name" field, and a "client key" and "client secret", located in the
 "Configuration" column.
-The client key and client secret values are the "App key" and ``App secret" fields which you saw earlier in your Dropbox app’s configuration settings page.
+The client key and client secret values are the "App key" and "App secret" fields which you saw earlier in your Dropbox app’s configuration settings page.
```

Comment:
shouldn´t hat be single backticks instead of quotes ?

Reply:
Not in this case, as it has a trailing/wrapping double-quote.